specs/paymentrequest.html

```diff
@@ -275,6 +275,15 @@
           list, the payment <code>details</code>, the payment <code>options</code>, and any <a>payment
           method</a> specific <code>data</code>.
         </p>
+
+        <p class="issue" data-number="40" title="How do developers layer additional information in the core payment messages?">
+          How does a developer access the <code>data</code> object?
+          How is <code>data</code> sent to the payment app?
+          Are the core set of payment messages (request/response) extensible?
+          If they are extensible, what is the extensibility mechanism?
+          Is the browser required to understand this extensibility mechanism?
+        </p>
+
```

Comment:
Also note the resolution of the group which included wording for this issue marker:
https://github.com/w3c/webpayments/issues/83

This is not appropriate wording while we have no extensibility guidance document but should be included as soon as we do.